nsiqcppstyle_exe.py

```diff
@@ -72,6 +72,10 @@ def Usage():
                 (target/filefilter.txt)
                 If you provide the file path (not a folder path) for the target,
                 -f option should be provided.
+  --filter-string=<filter string>
+                Semicolon-delimited string of valid filter file lines.  Enables specifying
+                the contents of a filter file without reading from the file system (e.g.,
```

Comment:
Bad message? Reading is allowed on RO FS

Reply:
Mea culpa :-).  Changed to:

```
  --filter-string=<filter string>
                Semicolon-delimited string of valid filter file lines.  Enables specifying
                the contents of a filter file without writing it to the file system (e.g.,
                in a read-only file system)
```

Done.

Reply:
`without needing to create one` might be more explicit and informative

Reply:
Done.